api/configuration/v1alpha1/kongpluginbinding_types.go

```diff
@@ -65,8 +65,9 @@ func (c *KongPluginBinding) SetConditions(conditions []metav1.Condition) {
 
 // KongPluginBindingSpec defines specification of a KongPluginBinding.
 type KongPluginBindingSpec struct {
-	// PluginReference is a reference to the KongPlugin or KongClusterPlugin resource. It is required
-	PluginReference PluginRef `json:"pluginRef"`
+	// PluginReference is a reference to the KongPlugin or KongClusterPlugin resource.
+	// +kubebuilder:validation:Required
+	PluginReference *PluginRef `json:"pluginRef"`
```

Comment:
Why make this a pointer if this is required? 🤔

Reply:
Good point. Moved back and added validation on name.